references.bib

```diff
@@ -197,6 +197,51 @@ @inproceedings{harrington2009scipy
 	url={http://conference.scipy.org/proceedings/scipy2009/paper_14/}
 }
 
+@article{astropy,
+   title={The Astropy Project: Building an Open-science Project and Status of the v2.0 Core Package},
+   volume={156},
+   ISSN={1538-3881},
+   url={http://dx.doi.org/10.3847/1538-3881/aabc4f},
+   DOI={10.3847/1538-3881/aabc4f},
+   number={3},
+   journal={The Astronomical Journal},
+   publisher={American Astronomical Society},
+   author={Price-Whelan, A. M. and Sipőcz, B. M. and Günther, H. M. and Lim, P. L. and Crawford, S. M. and Conseil, S. and Shupe, D. L. and Craig, M. W. and Dencheva, N. and et al.},
+   year={2018},
+   month={Aug},
+   pages={123}
+}
+
+@article{Goldbaum2018,
+  doi = {10.21105/joss.00809},
+  url = {https://doi.org/10.21105/joss.00809},
+  year  = {2018},
+  month = {aug},
+  publisher = {The Open Journal},
+  volume = {3},
+  number = {28},
+  pages = {809},
+  author = {Nathan J. Goldbaum and John A. ZuHone and Matthew J. Turk and Kacper Kowalik and Anna L. Rosen},
+  title = {unyt: Handle,  manipulate,  and convert data with units in Python},
+  journal = {Journal of Open Source Software}
+}
+
+@online{pint,
+  title = {Pint: Operate and manipulate physical quantities in Python}
+  author = {{Pint Developers}}
+  year = {2012},  % First release?
+  url = {https://github.com/hgrecco/pint},
+  urldate = {2019-12-17}
+  }
+
+@online{pyadolc,
+  title = {PYADOLC: Python Bindings to ADOL-C}
```

Comment:
Missing ,s at the end of the title and author fields.

Reply:
opst, should have compiled with references...